IRRing.h

```diff
@@ -0,0 +1,24 @@
+
+#ifndef ARDUINO_IRRING_H
+#define ARDUINO_IRRING_H
+#include <Arduino.h>
+
+class IRRing  
+{
+	private:
+	double angle;
+	double strength;
+	double offset;
+	unsigned long*current_time;

```

Comment:
why is this kept as a pointer??

Reply:
If current time is going to be handled by other process maybe its fine to keep it like this but dangerous and more even thinking in the embedded space, if you have an example of where this value is being updated/changed other than this class can you show it here?